src/pages/Messages.jsx

```diff
@@ -22,51 +20,32 @@ import { MessageButtonGroup, MessageFolder } from '../components/Messages';
  */
 const Messages = () => {
   localStorage.setItem('restorePath', '/messages');
-  const { podUrl } = useContext(SignedInUserContext);
-  const { session } = useSession();
+  const { updateMessageCountState } = useContext(MessageContext);
```

Comment:
Is this still needed? This value is directly computed from the inbox (and doesn't seem super expensive to calculate), so you should be able to store the computation locally and then you don't need the context at all here.

Reply:
That's true. I'll try out a commit for this